app/component/itinerary/ItineraryPage.js

```diff
@@ -1171,13 +1200,21 @@ export default function ItineraryPage(props, context) {
         walking={walkPlan?.edges?.length > 0}
         biking={bikePlan?.edges?.length > 0 || !!bikePublicPlan?.edges?.length}
         driving={
-          (settings.includeCarSuggestions && carPlan?.edges?.length > 0) ||
+          (settings.includeCarSuggestions &&
+            (carPlan?.edges?.length > 0 ||
+              (config.carBoardingModes?.FERRY !== undefined &&
+                carPublicPlan?.edges?.length))) ||
```

Comment:
Do not test FERRY. It is not necessary and also against idea of configurable boarding modes.

Reply:
FERRY checking is now removed. The idea behind checking for FERRY is that only in the configs that it is defined in, the UI will show ferries. Now there isn't a config field for removing carpublic itineraries in case they are found but not wanted.

Reply:
OK I will test this.  A more correct way would be testing if boardingModes is not empty, but I guess that is not needed. Code looks just  fine to me now.